### PR TITLE
New homepage implement

### PR DESCRIPTION
**Pull request by lauraGgit**
Pull in and update a bunch of the stuff that was done in #3334 with the new theming.

And close out #3406 .

## Summary Checklist
**Hero**
- [x] Text has line height 1.35 / `lh(3)`
- [x] Space above and below text is reduced to 5 units
- [x] Substitute image for new crop

**We support**
- [x] Text and image both take up 6 columns
- [x] Image persists on mobile (maybe)

**What 18F Can do**
- [x] Columns have ~2 units of margin on the right or there is a `grid-gap` on the container
- [x] Icon and text have 2 units of space between them. 

**Agencies**
- [x] Need to add 2 units of space after the `h2` and the list of agencies
- [x]  Link goes to the top of on the What We Deliver page (i.e. no anchor link) 
- [x] 3 units of space after each agency item, and line-height is 1.35/ `lh(3)`
- [x] The agencies are in an intentional order (alphabetical or otherwise)

**Blog Posts**
- [x] Border on Related Posts is Medium 
- [x] The section left aligned with the content on the page

**Mobile** 
- [x] Padding is consistent between sections, ~2 units 
- [x] Hero: Order of items is large text, image, smaller text + CtA
- [x] Hero: Text is next size down on mobile
- [x] What 18F can do: media blocks have 2 units of space on bottom
- [ ] What 18F can do: text wraps to next line on extra small screens (e.g. ~320px)

### CODE REVIEW

**Comment by lauraGgit** [comment]
@igorkorenfeld in the "We support your mission" do you have suggestions on how to get the image to be against the bottom of that div?

**Comment by igorkorenfeld** [comment]
@lauraGgit one idea is to move it to the CSS as a background image. I pushed up a branch to consider

File: pages/home.html [hunk 1]
```diff
@@ -35,29 +40,80 @@ <h2>
...
-        work with government. Federal, state, and local governments can partner
-        with 18F to deliver projects that fulfill their mission, stay within
-        budget, and use leading technology practices.
-      </h3>
-    </section>
-  </section>
+<section class="usa-section bg-base-lightest padding-y-0">
+  <div class="grid-container">
+    <div class="grid-row grid-gap">
+      <div class="tablet:grid-col-6 padding-y-2 tablet:padding-y-6">
```
Comment (igorkorenfeld) [inline pages/home.html:46]: An idea to try, we get rid of the `tablet:padding-y-6` here and target the parent `grid-row` (should be specific to this section, so maybe adding a class name?) and add a css rule `align-items: center`

Comment (lauraGgit) [reply pages/home.html:46]: I'll give it a shot

**Comment by lauraGgit** [comment]
broke out a separate pr for the agency list

**Review by igorkorenfeld — CHANGES_REQUESTED** [review]
This is looking good and really close!

Last items are 
1. The "We Support" image sizing, which we can pull out into a separate issue. 
2. Agencies `H2` needs 2 units of spacing below the heading. This can be rolled into #3411 
3. On tablet and mobile the "What 18F Can do", "Featured Projects", "Some Agencies", and "Recent Blog posts" sections have extra padding on the left, so they are not aligned with the header, Hero, We support, and Footer sections. We should remove this extra padding.

image

**Comment by lauraGgit** [comment]
@igorkorenfeld ok this should be good for review. @Dahianna also would love your eyes too!

**Comment by igorkorenfeld** [comment]
Padding on mobile looks great now!

There are still some spacing issues:
1. We should have 2 units spacing between an `h2` heading and the content
- There currently isn't any after the Agencies `h2`
- There is too much (seeing 36px) after the Related Blog posts
- On mobile Related Blog posts has additional spacing on top of that (52px total)

2. On mobile the collapsed columns need 3 units of the margin at the bottom. Currently they have 0 so the agency list items get very close together and make for uneven spacing 

image


## Summary Checklist
- [ ] Agencies has 2 units of space between `h2` and first list item
- [ ] Related blog posts has 2 units of space between `h2` and first blue bar
- [ ] On mobile Related blog posts has 2 units of space between `h2` and first blue bar
- [ ] On mobile there are 3 units of space between all agency list items

**Review by igorkorenfeld — APPROVED** [review]
This looks good to me! @Dahianna would you mind looking this over once more?

**Comment by Dahianna** [comment]
Yes, it looks great!!

**Pull request merged**